R/read_asc_PerkinElmer.R

```diff
@@ -60,3 +60,48 @@ read_asc_PerkinElmer <- function(file = stop("filename or connection needed"),
 # Unit tests -----------------------------------------------------------------
 
 # FIXME: add unit tests, if possible.
+hySpc.testthat::test(read_asc_PerkinElmer) <- function() {
+  context("read_asc_PerkinElmer")
+
+  path <- system.file("extdata/fileio/txt.PerkinElmer",package = "hySpc.read.txt")
+  flu_file <- paste0(path, c("/flu1.txt", "/flu2.txt",
+                             "/flu3.txt", "/flu4.txt", "/flu5.txt", "/flu6.txt"))
+
+  test_that("PerkinElmer .txt labels are correct", {
+    for(flu in flu_file){
+      expect_message(spc <- read_asc_PerkinElmer(flu))
+
+      expect_true(is.expression(spc@label$spc))
+      expect_true(is.expression(spc@label$.wavelength))
+      expect_equal(spc@label$filename, "filename")
+    }
+  })
+
+  test_that("PerkinElmer .txt spectra data", {
+    for(flu in flu_file){
+      expect_message(spc <- read_asc_PerkinElmer(flu))
+
+      expect_equal(colnames(spc@data$spc),
+                   c("405",  "405.5","406",  "406.5","407",  "407.5","408",  "408.5","409",  "409.5","410",  "410.5","411",  "411.5","412",  "412.5",
```

Comment:
Can we reduce this vector to single line `as.character(seq(from = 405, to = 495, by = .5))`?